### PR TITLE
fix: topic subscription

### PR DESCRIPTION
**Pull request by kamiya4047**
part of #29

### CODE REVIEW

File: lib/core/api.dart [hunk 1]
```diff
@@ -10,28 +10,32 @@ class ExpTechApi {
   ExpTechApi({this.apikey});
 
   Future<List<PartialEarthquakeReport>> getReportList({int limit = 20}) async {
-    final response = await http.get(
-        Uri.parse('https://lb-3.exptech.com.tw/api/v2/eq/report?limit=$limit'));
+    final response = await http.get(Uri.parse('https://lb-3.exptech.com.tw/api/v2/eq/report?limit=$limit'));
```
Comment (yayacat) [inline lib/core/api.dart:13]: URL needs to be changed

File: lib/core/api.dart [hunk 2]
```diff
@@ -10,28 +10,32 @@ class ExpTechApi {
...
-      throw Exception(
-          'The server returned a status code of ${response.statusCode}');
+      throw Exception('The server returned a status code of ${response.statusCode}');
     }
   }
 
   Future<EarthquakeReport> getReport(String id) async {
-    final response = await http
-        .get(Uri.parse('https://lb-3.exptech.com.tw/api/v2/eq/report/$id'));
+    final response = await http.get(Uri.parse('https://lb-3.exptech.com.tw/api/v2/eq/report/$id'));
```
Comment (yayacat) [inline lib/core/api.dart:23]: URL needs to be changed

**Review by yayacat — CHANGES_REQUESTED** [review]
URL needs to be changed

File: lib/core/api.dart (hunk 2 above)
Comment (kamiya10) [reply lib/core/api.dart:23]: 不在這個 PR 應該處理的範圍內

File: lib/core/api.dart (hunk 1 above)
Comment (kamiya10) [reply lib/core/api.dart:13]: 不在這個 PR 應該處理的範圍內

**Review by yayacat — APPROVED** [review]
LGTM

File: lib/core/api.dart (hunk 1 above)
Comment (yayacat) [reply lib/core/api.dart:13]: 好

File: lib/core/api.dart (hunk 2 above)
Comment (yayacat) [reply lib/core/api.dart:23]: 好

**Review by whes1015 — APPROVED** [review]

**Pull request merged**